### DISCUSSION TITLE
Provide markup examples and illustrations within elementSpec

### DISCUSSION THREAD

**Discussion by bwbohl** [music-encoding/music-encoding#1603] [Ideas]
Adding this from closed issue in music-encoding/music-encoding, cf. https://github.com/music-encoding/music-encoding/issues/222

**Comment by bwbohl** [comment]
As mentioned in music-encoding/music-encoding#222 and obvious from the guidelines repo being separated from the schema development it is questionable whether to include code examples in the schema definition, nevertheless the guidelines and/or specs should provide examples and potentially even more than in the text-flow.
Maybe the best way would be to run a query of all examples in the guidelines as well as over all https://github.com/music-encoding/sample-encodings
and display the results at the end of the respective element spec

**Comment by musicEnfanthen** [comment]
Moving to main repo

**Comment by musicEnfanthen** [comment]
Covnerting to discussion
